standards/iso/src/test/groovy/org/codice/countrycode/standards/iso/Iso3166StandardProviderSpec.groovy

```diff
@@ -19,6 +19,6 @@ class Iso3166StandardProviderSpec extends Specification {
 
     def 'test iso3166 provider'() {
         expect:
-        new Iso3166StandardProvider().getStandardEntries().size() == 257
```

Comment:
we should delete this test

Reply:
i'm not sure about that; this forces us to think about augmenting the json